.gitattributes

```diff
@@ -1,12 +1,15 @@
 /tests export-ignore
```

Comment:
Let's sort everything alphabetically. It's confusing to see `phpunit.xml` ignored but not `tests/` because tests go first.

Reply:
:+1: